packages/store-ui/src/Carousel/Page.tsx

```diff
@@ -14,18 +15,24 @@ interface Props {
   height: string
   loading: 'eager' | 'lazy'
   variant: string
+  link?: ComponentType
 }
 
-const CarouselPage: FC<Props> = ({ item, loading, width, height, variant }) => (
-  <LocalizedLink key={item.href} to={item.href}>
+export const CarouselPage: FC<Props> = ({
+  item,
+  loading,
+  width,
+  height,
+  variant,
+  link: Link = LocalizedLink,
```

Comment:
please use the normal Gatsby link as default in here so stores without localization don't pay the burden of the localized link